signingscript/utils.py

```diff
@@ -167,3 +167,13 @@ def is_autograph_signing_format(format_):
 
     """
     return format_ and format_.startswith('autograph_')
+
+
+def is_apk_autograph_signing_format(format_):
+    """Return bool of whether a signing format is an APK.
+
+    Args:
+        format_ (str): the format to check
+
+    """
+    return format_ and format_ in ('autograph_fennec', 'autograph_focus')
```

Comment:
Hm, potential dupe with https://github.com/mozilla-releng/signingscript/pull/88/files#diff-c18cb9a0544a9707c29e2a2cf9d2d23eR162 ? I suppose above refers to `mar` and others as well but just wanted to check it's intended. If we keep this, we'll have to spin up another condition for when we'll publish the `reference-brower` or potentially `android-components`. But it's definitely not blocking for now.

Reply:
Yeah, that's intended.

I agree we should define a nomenclature that avoids to list every format here. Let's do it in a follow up.

Reply:
Sounds good.